TomeOfKnowledge.playground/contents.xcplayground

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<playground version='6.0' target-platform='osx' requires-full-environment='true' display-mode='rendered'>
+<playground version='6.0' target-platform='osx' requires-full-environment='true' display-mode='raw'>
```

Comment:
Hmm... not sure what this is? I'll take a peek at it. Oh, that must be because I committed with the raw markup viewed?